cosmo_tester/framework/testenv.py

```diff
@@ -265,6 +265,13 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def wait_for_expected_outputs(self, expected_outputs, timeout=3):
```

Comment:
the other places I saw getting outputs had delays. but those were monitoring. I guess it can be removed if you think it is a bad idea.